apps/wallet/src/ui/app/pages/home/nft-details/LabelValueItem.tsx

```diff
@@ -49,7 +49,7 @@ export function LabelValueItem({
     }
     return display ? (
         <div className="flex flex-row flex-nowrap gap-1">
-            <div className="flex-1 overflow-hidden">
+            <div className="flex-1 overflow-hidden [&>div]:leading-5">
```

Comment:
I think if we update the `Text`, `Link`, (or typography in general) etc to have line-height to something line `1.2` and not `1` will fix it and we will avoid targeting specific divs etc with custom selectors

Reply:
I agree, but the issue is also that there will be increased spacing wherever the component is used and it seems like this is the only instances where there is an issue created due to the truncating

Reply:
Just came her to chime in that let's not update the typography in general until i have had more time to research it. 🙏🏽

Reply:
I'm happy to make this a one-off and revisit later

Reply:
Ahh you're right - I will just hold off here until abhi makes a decision then!